proto/src/main/proto/Rss.proto

```diff
@@ -321,3 +324,12 @@ message CheckServiceAvailableResponse {
   StatusCode status = 1;
   bool available = 2;
 }
+
+message AccessClusterRequest {
+  string cronTaskParam = 1;
```

Comment:
cronTaskParam is too specific, how about accessInfo?

Reply:
OK